### DISCUSSION TITLE
How to let renovatebot know, about a value of variable being used in gradle.

### DISCUSSION THREAD

**Discussion by gsbindhani** [renovatebot/renovate#31470] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure Devops

### Please tell us more about your question or problem

Hi, 
Thank you for taking a look into my question and I appreciate your time.
I'm fairly new to renovate bot and trying to find a solution to the below problem.

Here is my case. I have a Java gradle project and in the gradle file, we have something like this.
_implementation group: "org.springframework.boot', name: 'spring-boot', **version: '${spring_boot_version}"**_

Here, the version of **spring_boot_version** is getting passed at the pipeline gradle task and to build the code, we are using the below command and passing the exact value of boot version.

**./gradlew clean build --project-prop spring_boot_version=3.3.3**

Now, renvovate bot does not know about the value and its trying to resolve the variable **${spring_boot_version}** from gradle.properties and we kept the varaible with some random value. Its working and reading the value from gradle.properties.

But, how we can configure the renovate bot, to let it know that, if any gradle file has a variable  **spring_boot_version**, use the value as X.

**Comment by RahulGautamSingh** [comment]
Not possible through renovate config.
